src/applications/mhv-landing-page/components/NavCard.jsx

```diff
@@ -4,7 +4,12 @@ import recordEvent from '~/platform/monitoring/record-event';
 
 export const externalLinkText = '(opens in new tab)';
 
-const NavCard = ({ icon = null, title, links }) => {
+const NavCard = ({
+  icon = null,
+  iconClasses = 'vads-u-margin-right--1p5',
```

Comment:
Is there a need to pass in props for icon classes?

Reply:
The `attach_money` icon needed some visual adjustment due to it being a narrow piece of artwork. I figured passing iconClasses in was a reasonable way to adjust the one icon that needed it